docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -265,6 +265,7 @@ How to **investigate**:
 - If the failing service is going OOM (`OOMKilled`): scale up or increase the memory
 - If the failing service is crashing / panicking: look for the stack trace in the logs and investigate from there
   - If crashing service is query-frontend, querier or store-gateway, and you have "activity tracker" feature enabled, look for `found unfinished activities from previous run` message and subsequent `activity` messages in the log file to see which queries caused the crash.
+- When using Memberlist as KV store for hash rings, ensure that Memberlist is working correctly. See [instructions for `MimirGossipMembersMismatch` alert.](#MimirGossipMembersMismatch)
```

Comment:
I wouldn't link the ending dot. Also i think @grafana/docs-squad wants a link to match the destination text, so only link `MimirGossipMembersMismatch` (at least that's what I understood...):

```suggestion
- When using Memberlist as KV store for hash rings, ensure that Memberlist is working correctly. See instructions for [`MimirGossipMembersMismatch`](#MimirGossipMembersMismatch) alert.
```

_Same comment applies to other places._

Reply:
Applied everywhere.